### PR TITLE
2316 - Pop up error message

### LINKED ISSUE #2316: pop up Error message ODP
Capture
should include small error message on the highlighted area, no?

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #2316

https://user-images.githubusercontent.com/10047945/222120327-c6064c11-2cab-4a26-8d05-f122dba03073.mov

### CODE REVIEW

File: src/i18n/resources/en.js [hunk 1]
```diff
@@ -1050,6 +1050,7 @@ The FRA team
       'The sum of the reported FRA categories exceeds the forest area reported in table 1a',
     valueEqualToSum: 'Total value should be equal to sum of subcategories',
     landAreaExceedsTotalLandArea: 'Land area exceeds total land area (1a)',
+    classValueNotGreaterThan: '{{name}} should be not greater than {{value}}',
```
Comment (sorja) [inline src/i18n/resources/en.js:1053]: `{{value}} %`

also can be directly 100%? it's always 100

Comment (sorja) [reply src/i18n/resources/en.js:1053]: In the video I noticed that 're' is only 10% but error shows 'greater than 100'
Should it be: class should total to 100%?

Comment (rfontanarosa) [reply src/i18n/resources/en.js:1053]: I would like to keep it dynamic, sooner or later I would like reduce all the error messages to a small bunch of dynamic ones.

Screenshot 2023-03-01 alle 13 40 13

File: src/client/pages/OriginalDataPoint/components/ForestCharacteristics/ForestCharacteristicsPlantation.tsx [hunk 2]
```diff
@@ -65,6 +75,20 @@ const ForestCharacteristicsPlantation: React.FC<Props> = (props) => {
             </tr>
           </tfoot>
         </table>
+
+        {hasErrors && (
```
Comment (minotogna) [inline src/client/pages/OriginalDataPoint/components/ForestCharacteristics/ForestCharacteristicsPlantation.tsx]: I saw this several times, is it possible to have a common component ?

File: src/i18n/resources/en.js (hunk 1 above)
Comment (sorja) [reply src/i18n/resources/en.js:1053]: 'to 100%'

**Review by minotogna — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**